portal/docs/changelog.md

```diff
@@ -4,16 +4,19 @@ sidebar_position: 1
 
 # Change Log
 
-** Keep track of every change to the Kiva Partner API. **
```

Comment:
I'm splitting hairs here, but I don't _quite_ want to commit to updating this for every change 😅 I'd like to keep it isolated to pertinent partner-facing changes. Open to other ideas/wordings.

Reply:
agreed.   public affecting changes should be published (aka bug fixes, new api, changed api (although I hope we version it).   Maybe publishing info on changes if we have concerns about a change such as performance